docs/modules/integrate/pages/legacy-cdc-connectors.adoc

```diff
@@ -0,0 +1,98 @@
+= Legacy CDC Connector
+
+Change Data Capture (CDC) refers to the process of observing changes
+made to a database and extracting them in a form usable by other
+systems, for the purposes of replication, analysis and many more.
+
+Change Data Capture is especially important to Hazelcast, because it allows
+for the _streaming of changes from databases_, which can be efficiently
+processed by the Jet engine.
+
+Implementation of CDC in Hazelcast is based on
+link:https://debezium.io/[Debezium]. Hazelcast offers a generic Debezium source
+which can handle CDC events from link:https://debezium.io/documentation/reference/stable/connectors/index.html[any database supported by Debezium],
+but we're also striving to make CDC sources first class citizens in Hazelcast.
+The ones for MySQL and PostgreSQL already are.
+
+== Installing the Connector
+
+This connector is included in the full distribution of Open Source Hazelcast.
+
+== CDC as a Source
+
+We have the following types of CDC sources:
+
+* link:https://docs.hazelcast.org/docs/{full-version}/javadoc/com/hazelcast/jet/cdc/DebeziumCdcSources.html[DebeziumCdcSources]:
+  generic source for all databases supported by Debezium
+* link:https://docs.hazelcast.org/docs/{full-version}/javadoc/com/hazelcast/jet/cdc/mysql/MySqlCdcSources.html[MySqlCdcSources]:
+  specific, first class Jet CDC source for MySQL databases (also based
+  on Debezium, but benefiting the full range of convenience Jet can
+  additionally provide)
+* link:https://docs.hazelcast.org/docs/{full-version}/javadoc/com/hazelcast/jet/cdc/postgres/PostgresCdcSources.html[PostgresCdcSources]:
+  specific, first class CDC source for PostgreSQL databases (also based
+  on Debezium, but benefiting the full range of convenience Hazelcast can
+  additionally provide)
+
+For the setting up a streaming source of CDC data is just the matter of pointing it at the right database via configuration:
+
+```java
+Pipeline pipeline = Pipeline.create();
+pipeline.readFrom(
+    MySqlCdcSources.mysql("customers")
+            .setDatabaseAddress("127.0.0.1")
+            .setDatabasePort(3306)
+            .setDatabaseUser("debezium")
+            .setDatabasePassword("dbz")
+            .setClusterName("dbserver1")
+            .setDatabaseWhitelist("inventory")
+            .setTableWhitelist("inventory.customers")
+            .build())
+    .withNativeTimestamps(0)
+    .writeTo(Sinks.logger());
+```
+
+For an example of how to use CDC data see xref:pipelines:cdc.adoc[our tutorial].
```

Comment:
Just a question; the same tutorial is referenced from both the Enterprise & legacy pages, in all likelihood it will be updated for the EE connector. 

I'd suggest that you consider whether to remove the link from this file or whether we also need to do something with the tutorial to provide one for each implementation.

Reply:
The API should be almost the same (this was one of main goals - old apps should work with very little to no changes), so maybe in parts where there are differences I will add tabs OS / EE?

Reply:
a25adae0ef13f4769a4ae54a5245bd9525997659

What do you think?

Reply:
Perfect